mariadb_kernel/maria_magics/load.py

```diff
@@ -0,0 +1,72 @@
+"""This class implements the %load magic command"""
+
+help_text = """
+The %load magic command has the following syntax:
+    > %load [csv file path] [table name] [skip row num](optional)
+The %load magic command can load CSV file for updating specific table data.
+1. This command does not create a table if the one specified as argument doesn't exist,
+   the user needs to create the destination table with the proper schema to match the data in the CSV file.
+2. CSV file first line may be header, can set [skip row num] to 1 for skipping.
+3. any argument can be enclosed by '' or "", handling string contains spaces.
+"""
+
+from mariadb_kernel.maria_magics.line_magic import LineMagic
+import pandas
+import shlex
+
+
+class Load(LineMagic):
+    def __init__(self, args):
+        args_list = shlex.split(args)
+        self.csv_file_path = ""
+        self.table_name = ""
+        self.skip_row_num = 0
+        if len(args_list) == 2:
+            self.csv_file_path = args_list[0]
+            self.table_name = args_list[1]
+        elif len(args_list) == 3:
+            self.csv_file_path = args_list[0]
+            self.table_name = args_list[1]
+            self.skip_row_num = args_list[2]
+
+    def name(self):
+        return "%load"
+
+    def help(self):
+        return help_text
+
+    def execute(self, kernel, data):
+        # for error handling
```

Comment:
This comment is too vague and doesn't help the developer reading it too much. Either remove it or replace it with a text that explains what happens in the code below it.

Reply:
I remove it.